cvat/settings/base.py

```diff
@@ -413,6 +420,9 @@ class CVAT_QUEUES(Enum):
 CACHE_ROOT = os.path.join(DATA_ROOT, 'cache')
 os.makedirs(CACHE_ROOT, exist_ok=True)
 
+EXPORT_CACHE_ROOT = os.path.join(CACHE_ROOT, 'export')
+os.makedirs(EXPORT_CACHE_ROOT, exist_ok=True)
```

Comment:
This declaration should be duplicated in testing settings, otherwise the real local cache dir will be used in tests.

Reply:
Thanks, I forgot about this